stetho-repl-js/README.md

```diff
@@ -0,0 +1,223 @@
+# Stetho's Javascript Console
+
+This [Stetho](https://facebook.github.io/stetho) plugin adds a javascript console by embedding Mozilla's [Rhino](https://github.com/mozilla/rhino).
+
+## Set-up
+
+### Download
+Download [the latest JARs](https://github.com/facebook/stetho/releases/latest) or grab via Gradle:
+```groovy
+compile 'com.facebook.stetho:stetho-repl-js:1.1.1'
+```
+or Maven:
+```xml
+<dependency>
+  <groupId>com.facebook.stetho</groupId>
+  <artifactId>stetho-repl-js</artifactId>
+  <version>1.1.1</version>
+</dependency>
+```
+
+Make sure that you depend on the main `stetho` dependency too.
+
+### Putting it together
+The javascript integration is similar to standard Stetho integration.
+The main difference is with the WebKitInspector used.
+There is a simple initialization step which occurs in your `Application` class:
+
+```java
+public class MyApplication extends Application {
+  public void onCreate() {
+    super.onCreate();
+    JavaScriptRuntimeBuilder jsRuntimeBuilder = new JavaScriptRuntimeBuilder(this);
```

Comment:
From what I've read there's not that much that we can do in code blocks besides selecting the language. I'll try to see if I can use a table with the original block on the left and the modified block on the right.

If it would work i could mimic the output of sdiff and it will help to understand what was added.